templates/header.hbs

```diff
@@ -9,15 +9,20 @@
       {{/if}}
     {{/link}}
   </div>
-  <div class="nav-wrapper">
-    <button class="menu-button" aria-controls="user-nav" aria-expanded="false" aria-label="{{t 'toggle_navigation'}}">
-      <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" focusable="false" viewBox="0 0 16 16" class="icon-menu">
-        <path fill="none" stroke="currentColor" stroke-linecap="round" d="M1.5 3.5h13m-13 4h13m-13 4h13"/>
-      </svg>
-    </button>
+
+  <div class="nav-wrapper-desktop">
     <nav class="user-nav" id="user-nav">
-      {{link 'community'}}
-      {{link 'new_request' class='submit-a-request'}}
+      <ul class="user-nav-list">
+        <li>{{link 'community'}}</li>
+        <li>{{link 'new_request' class='submit-a-request'}}</li>
+        {{#unless signed_in}}
+          <li>
+            {{#link "sign_in" class="sign-in"}}
+              {{t 'sign_in'}}
+            {{/link}}
+          </li>
+        {{/unless}}
+      </ul>
     </nav>
```

Comment:
Well, ideally probably this should be in a nav item, but I've decided that's out of the scope of this PR (I've talked about that with Luis a couple of weeks ago and he agreed). This PR got pretty big anyway 😅 

As a reminder - initially, this was just a small task to put [those links](https://github.com/zendesk/copenhagen_theme/blob/master/templates/header.hbs#L19-L20) in a list because that got flagged during A11y review.

To sum up - we have only 1 week left to ship it, I don't think adding even more improvements is a good idea in this situation. I suggest we put this improvement in our backlog if we think that's important though.